tests/Feature/FileAdder/IntegrationTest.php

```diff
@@ -296,10 +296,22 @@ public function it_can_add_a_remote_file_to_the_medialibrary()
     }
 
     /** @test */
-    public function it_can_add_a_remote_file_with_no_name_to_the_medialibrary()
+    public function it_can_add_a_remote_file_with_the_name_of_the_last_directory_to_the_medialibrary()
     {
         $url = 'https://docs.spatie.be/laravel-medialibrary/test-image';
 
+        $media = $this->testModel
+            ->addMediaFromUrl($url)
+            ->toMediaCollection();
+
+        $this->assertFileExists($this->getMediaDirectory("{$media->id}/test-image.jpeg"));
+    }
+
+    /** @test */
+    public function it_can_add_a_remote_file_with_no_name_or_directory_to_the_medialibrary()
```

Comment:
This test now fails: https://travis-ci.org/spatie/laravel-medialibrary/jobs/356919799

Reply:
Yes the test fail because the PR at the docs site is not yet merged. After that it will pass.